cas-server-support-oauth/src/main/java/org/jasig/cas/support/oauth/profile/OAuthUserProfile.java

```diff
@@ -0,0 +1,14 @@
+package org.jasig.cas.support.oauth.profile;
+
+import org.pac4j.http.profile.HttpProfile;
+
+/**
+ * Specific profile for OAuth user authentication.
+ *
+ * @author Jerome Leleu
+ * @since 4.3.0
+ */
+public class OAuthUserProfile extends HttpProfile {
```

Comment:
Good question. I have special profile types because I want to be sure that the profiles have been created by the appropriate authenticators to avoid any vulnerability coming from cross authentication: using the client grant and authenticating as a user for example...